context-free.md

```diff
@@ -2,15 +2,19 @@
 
 ##  Who is this programming language for?
 
+Context Free is essentially a computer programming language with potential utilisation in graphic design owing to its ability to produce intricate images in terms of variations of simple shapes: squares, triangles, and circles. A few lines of code of skewing, rotating, transforming and flipping these basic shapes as well as using complex programming concepts such as loops and recursion enables highly-detailed, organised, and specific designs to be created. In my view,  this language is used to serve those users who wish to create intricate and detailed patterns such as fractals, since these are easier to generate with a greater level of accuracy through programming statements rather than manually using Photoshop or another photo editor.
 
 ## What is easy to do in this language? Why is it easy?
 
+Context Free is a Domain-Specific Language whose limited-expressiveness in terms of its methods and parameters makes it ideal for generating and creating complex patterns and images. It contains parameters such as chirality, transformation, rotation, flipping which create domain focus on image creation since these concepts are all easily-applicable and mostly-specific to images. At the same time, these are concepts comprehensible by most regardless of their experience in graphic design thus increasing the usability of the language to the general public and anybody willing and able to learn it. Its limited-expressiveness in terms of the fact that it defines images only through certain primitive shapes and a few shape and colour readjustment methods means that it can easily be learnt and used by beginners. Generating detailed and patterned images such as fractals is also much easier and more convenient to do using this language as compared to using a typical photo editor.
```

Comment:
I agree with the definition you made for Context Free. I also think Context Free makes everything seem simpler and easier for beginners.